tools/oneclickapps/prepare_digital_ocean_one_click_app_release.py

```diff
@@ -0,0 +1,158 @@
+import os
+import subprocess
+import time
+from pathlib import Path
+from typing import List
+
+import digitalocean
+import zulip
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+manager = digitalocean.Manager(token=os.environ["DIGITALOCEAN_API_KEY"])
+zulip_client = zulip.Client()
+TEST_DROPLET_SUBDOMAIN = "oneclicktest"
+
+
+def generate_ssh_keys() -> None:
+    subprocess.call(
+        ["ssh-keygen", "-f", str(Path.home()) + "/.ssh/id_rsa", "-P", "", "-t", "ed25519"]
```

Comment:
We don't want to name it `id_rsa`:
```suggestion
        ["ssh-keygen", "-f", str(Path.home()) + "/.ssh/id_ed25519", "-P", "", "-t", "ed25519"]
```